lib/cc/engine/analyzable_files.rb

```diff
@@ -17,7 +17,7 @@ def all
 
       def filter_files(files)
         files.select do |file|
-          file.end_with?("/") || file.end_with?(".coffee")
+          file.end_with?("/") || file.end_with?(".coffee") || file.end_with?(".coffee.md") || file.end_with?(".litcoffee")
```

Comment:
@KrysKruk seems like this is begging to be DRYed up. What do you think of the following?

```ruby
SUFFIXES = %w[/ .coffee .coffee.md .litcoffee].freeze

...

# in filter_files
SUFFIXES.any? { |suffix| file.end_with?(suffix) }
```